src/components/Table/helpers/columns/__test__/isNotSameColumns.spec.js

```diff
@@ -7,18 +7,104 @@ describe('isNotSameColumns', () => {
         expect(isNotSameColumns(prevColumns, currentColumns)).toBe(true);
     });
     it('should return false when prevColumns and currentColumns are the same', () => {
-        const prevColumns = [{ field: 'name', label: 'Name' }, { field: 'email', label: 'Email' }];
+        const prevColumns = [
+            { field: 'name', header: 'Name' },
+            { field: 'email', header: 'Email' },
+        ];
         const currentColumns = [
-            { field: 'name', label: 'Name' },
-            { field: 'email', label: 'Email' },
+            { field: 'name', header: 'Name' },
+            { field: 'email', header: 'Email' },
         ];
         expect(isNotSameColumns(prevColumns, currentColumns)).toBe(false);
     });
-    it('should return true when prevColumns and currentColumns are the different', () => {
-        const prevColumns = [{ field: 'name', label: 'Name' }, { field: 'email', label: 'Email' }];
+    it('should return true when column field property changed', () => {
+        const prevColumns = [
+            {
+                field: 'name',
+                header: 'Name',
+                component: {},
+                type: 'string',
+                defaultWidth: 100,
+                width: 'auto',
+            },
+            {
+                field: 'email',
+                header: 'Email',
+                component: {},
+                type: 'string',
+                defaultWidth: 100,
+                width: 'auto',
+            },
+        ];
+        const currentColumns = [
+            { field: 'phone', header: 'Name' },
+            { field: 'email', header: 'Email' },
+        ];
+        expect(isNotSameColumns(prevColumns, currentColumns)).toBe(true);
```

Comment:
I said that remove this because it have other props prevColumns not present in currentColumns then there are others props that will cause the function to return true so it is not exact
Then remove this and leave the one removed that is similar to below tests with:
```
const prevColumns = [
            { field: 'name', header: 'Name' },
            { field: 'email', header: 'Email' },
        ];
        const currentColumns = [
            { field: 'phone', header: 'Name' },
            { field: 'email', header: 'Email' },
        ];
```